### DISCUSSION TITLE
Any suggestion on how I can incorparate audio in next js

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [vercel/next.js#31270] [Help]
How can one have an audio play in a website using next JS

**Comment by delbaoliveira** [comment]
Hi @KakoozaJerry,

This answer in particular might be helpful, and the discussion in general: https://github.com/vercel/next.js/discussions/12810#discussioncomment-1297794

**Reply by KakoozaJerry** [reply]
Hello @delbaoliveira , thanks for the response. Is there any audio component that one may use and they plug in the src of the audio as a url in next JS.

Something like
`<audio src="https://bensound.mp3> </audio>`

Thank you
